eslint.dogfooding.config.js

```diff
@@ -55,6 +55,14 @@ const config = [
 			'unicorn/prefer-module': 'off',
 		},
 	},
+	{
+		plugins: {
+			jsdoc,
+		},
+		rules: {
+			'jsdoc/require-asterisk-prefix': ['error', 'never', {}],
+		},
+	},
```

Comment:
It should be added to `eslint.config.js`, not this file.

Reply:
Updated